pyOpenMS/manylinux/dockerfile

```diff
@@ -0,0 +1,36 @@
+## openms/manylinux is based on hroest/manylinux2014_qt59:v1.1
+ARG CONTRIB_BRANCH_OR_TAG=${CONTRIB_BRANCH_OR_TAG}
+
+FROM quay.io/pypa/manylinux2014_x86_64
+
+RUN yum install -y wget
+RUN yum install -y xz qt5-qtbase-devel 
+RUN yum install -y qt5-qtbase-devel qt5-qtsvg qt5-qtsvg-devel python-devel 
+RUN yum install -y libsvm-devel glpk-devel libzip-devel zlib-devel xerces-c-devel bzip2-devel sqlite-devel hdf5-devel
+RUN yum install -y libtool
+
+RUN yum install -y cmake3
+RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
+
+# install contrib (need ALL here to build static libs)
+RUN git clone -b ${CONTRIB_BRANCH_OR_TAG} --single-branch https://github.com/OpenMS/contrib.git && \
+    mkdir contrib-build && \
+    cd /contrib-build && \
+    cmake -DBUILD_TYPE=ALL ../contrib && \
+    rm -rf archives src && \
+    cd / && \
+    rm -rf contrib
+
+WORKDIR /
+
+# Metadata
+LABEL base.image="manylinux2014_x86_64"
+LABEL version="1.0"
+LABEL software="OpenMS (dependencies for manylinux2010)"
+LABEL software.version="qt5"
```

Comment:
Ah sorry, with "qt5 is the default", I meant that you can remove it as a modifier to the version. Can you use the version from the hook in the LABEL?
I.e. in the hook: if  branch = release/* then version = * else version = nightly

Reply:
ok, no problem. This is only the contrib, so should the version should probably be based on the branch/tag used, or? 
`master`
`Release 2.6.0`
So could you just use the variable (`CONTRIB_BRANCH_OR_TAG`)  for the label? Or was that what you meant?